backend/.env.bak

```
@@ -0,0 +1,164 @@
+ENVIRONMENT=development
+
+# Alembic auto-upgrade configuration
+# Enable/disable automatic database migration on startup (only works in development mode)
+DB_AUTO_MIGRATE=True
+
+# Database configuration
+DATABASE_URL=mysql+pymysql://root:123456@localhost:3306/task_manager
+
+# JWT configuration
+SECRET_KEY=your-secret-key-here
+ALGORITHM=HS256
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The ALGORITHM key should go before the SECRET_KEY key

(UnorderedKey)